src/main/java/com/xceptance/xlt/report/util/JFreeChartUtils.java

```diff
@@ -1508,34 +1543,6 @@ public static void adjustSeriesForLogarithmicAxes(final XYIntervalSeries interva
         }
     }
 
-    /**
-     * Sets the compression level to use when creating PNG images.
-     *
-     * @param level
-     *            the compression level
-     */
-    public static void setPngCompressionLevel(final int level)
-    {
-        if (0 <= level && level <= 9)
-        {
-            pngCompressionLevel = level;
-        }
-        else
-        {
-            throw new IllegalArgumentException("The PNG compression level must be between 0...9");
-        }
-    }
-
-    /**
-     * Returns the compression level to use when creating PNG images.
-     *
-     * @return the compression level
-     */
-    public static int getPngCompressionLevel()
-    {
-        return pngCompressionLevel;
-    }
```

Comment:
I would like to keep the opportunity to set the desired compression level/quality, in modified form of course, i.e. as data type double with 0.75 as the default. Please rework `ReportGeneratorConfiguration` and `config/reportgenerator.properties` accordingly.

Reply:
Done